eng/liveBuilds.targets

```diff
@@ -214,6 +216,22 @@
         $(LibrariesNativeArtifactsPath)dotnet.native.js.symbols;
         $(LibrariesNativeArtifactsPath)*.dat;"
         IsNative="true" />
+      
+      <LibrariesRuntimeFiles Condition="'$(TargetOS)' == 'browser' and '$(RuntimeFlavor)' == 'CoreCLR'"
+                             Include="
+        $(CoreCLRNativeArtifactsPath)dotnet.js;
+        $(CoreCLRNativeArtifactsPath)dotnet.js.map;
+        $(CoreCLRNativeArtifactsPath)dotnet.runtime.js;
+        $(CoreCLRNativeArtifactsPath)dotnet.runtime.js.map;
+        $(CoreCLRNativeArtifactsPath)dotnet.d.ts;
+        $(CoreCLRNativeArtifactsPath)dotnet-legacy.d.ts;
+        $(CoreCLRNativeArtifactsPath)package.json;"
+        IsNative="true" />
+      <LibrariesRuntimeFiles Condition="'$(TargetOS)' == 'browser' and '$(RuntimeFlavor)' == 'CoreCLR'"
+                             Include="$(CoreCLRNativeArtifactsPath)src\es6\*.js"
+                             NativeSubDirectory="src\es6"
+                             IsNative="true" />
```

Comment:
This treats the runtime JS as "libraries files", which works, and is fine for now, but is not ideal, since these are, after all, are "runtime" files. Could you add an item of "move runtime JS files to `/aotsdk` instead of `/framework`" to #2434?